.soliumrc.json

```diff
@@ -8,6 +8,8 @@
       "error",
       "double"
     ],
+    "security/no-inline-assembly": ["warning"],
```

Comment:
Can you explain this change in the PR description please?

Reply:
Will update the description and create an issue to track